src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImageFunctionalTest.groovy

```diff
@@ -73,7 +73,7 @@ class DockerSaveImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
 
             task buildImage(type: DockerBuildImage) {
                 dependsOn dockerfile
-                inputDir = file("build/docker")
+                inputDir = file("${BUILD_DIR}")
                 tags.add("${IMAGE_ID}")
```

Comment:
When tag does not contain repository (e.g. alpine:foo) then the string is treated as repository and the tag is always "latest".

Reply:
I don't fully understand why this had to change just because we bumped up the Docker Java version. Can you explain in more detail?

Reply:
Can you please explain this change?

Reply:
@mikealdo Can you explain why this needed to change. I am not going to merge the pull request before we had a chat about it.